### PR TITLE
output modifier

### PR DESCRIPTION
**Pull request by martinadamik**
## TODO
- [ ] test with custom mutation

### CODE REVIEW

File: filejet-external-php/src/ReplaceHtml.php [hunk 1]
```diff
@@ -79,18 +79,52 @@ private function replaceImageTags()
...
+            $srcSet = $image->getAttribute('srcset');
+            if(empty($srcSet)) {
+                continue;
+            }
+
+            $sources = explode(', ', $srcSet);
+            $newSources = [];
+
+            foreach($sources as $source) {
+                [$url, $w] = explode(' ', $source);
```
Comment (ivanbarlog) [inline filejet-external-php/src/ReplaceHtml.php]: Is this compatible with PHP 7.0?

File: filejet-external-php/src/ReplaceHtml.php [hunk 2]
```diff
@@ -79,18 +79,52 @@ private function replaceImageTags()
...
+                $newSources[] = "$newUrl $w";
+
+            }
+            
+            $image->setAttribute('srcset', implode(', ', $newSources));
         }
     }
 
+    public function getAspectRatio(int $width, int $height): float {
+        $gcd = function($width, $height) use (&$gcd) {
```
Comment (ivanbarlog) [inline filejet-external-php/src/ReplaceHtml.php]: what `$gcd` stands for?

File: filejet-external-php/src/ReplaceHtml.php [hunk 3]
```diff
@@ -79,18 +79,52 @@ private function replaceImageTags()
...
+            return ($width % $height) ? $gcd($height, $width % $height) : $height;
+        };
+
+        $g = $gcd($width, $height);
```
Comment (ivanbarlog) [inline filejet-external-php/src/ReplaceHtml.php]: what `$g` stands for?

**Review by ivanbarlog — CHANGES_REQUESTED** [review]

**Comment by ivanbarlog** [comment]
Also, please do squash the commits. Thanks

File: filejet-external-php/src/ReplaceHtml.php (hunk 2 above)
Comment (martinadamik) [reply filejet-external-php/src/ReplaceHtml.php]: greatest common divisor

**Review by ivanbarlog — APPROVED** [review]

**Pull request merged**